template/conftest.py

```diff
@@ -7,8 +7,10 @@
 
 
 # This variable is needed for Speculos only (physical tests need the application to be already installed)
+# Adapt this path to your 'tests/elfs' directory
 APPS_DIRECTORY = (Path(__file__).parent.parent / "elfs").resolve()
 
+# Adapt this path to the APPNAME in your Makefile
```

Comment:
Not really the APPNAME of the Makefile, but rather the `<name>_<device>.elf` used to store ELF binaries.